packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -443,7 +443,14 @@ export namespace AbstractVSXExtensionComponent {
 
 const downloadFormatter = new Intl.NumberFormat();
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const downloadCompactFormatter = new Intl.NumberFormat(undefined, { notation: 'compact', compactDisplay: 'short' } as any);
+const downloadCompactFormatter = new Intl.NumberFormat('en-US', { notation: 'compact', compactDisplay: 'short' } as any);
+const averageRatingFormatter = (averageRating: number): number => Math.round(averageRating * 2) / 2;
+const getAverageRatingTitle = (averageRating: number | undefined): string => {
```

Comment:
Is there a case where `getAverageRatingTitle` is called where `averageRating` is `undefined`?

Reply:
This case that I considered before will not exist and has been removed.